### PR TITLE
Add CommandLine exes: PlayerDataReset, XblDevAccount. Add token caches. Update Auth APIs.

### PR DESCRIPTION
**Pull request by jicailiu**

### CODE REVIEW

File: CommandLine/XblDevAccount/Program.cs [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            Console.WriteLine($"{indent}AccountSource : {devAccount.AccountSource}");
+        }
+
+        private static void OnSignOut()
+        {
+            DevAccount account = Auth.LoadLastSignedInUser();
+            if (account != null)
+            {
+                Auth.SignOut();
+                Console.WriteLine($"Developer account {account.Name} from { account.AccountSource} has successfully signed out. ");
```
Comment (jasonsandlin) [inline CommandLine/XblDevAccount/Program.cs]: nit: extra space after {

Comment (jasonsandlin) [reply CommandLine/XblDevAccount/Program.cs]: and below

File: CommandLine/XblDevAccount/Program.cs [hunk 2]
```diff
@@ -0,0 +1,173 @@
+﻿using System.Net;
```
Comment (jasonsandlin) [inline CommandLine/XblDevAccount/Program.cs]: add copyright lines here and elsewhere

Comment (I-Cat) [reply CommandLine/XblDevAccount/Program.cs]: Your copyright is inccorect

**Pull request merged**